src/scheduler/round_robin.rs

```diff
@@ -7,23 +7,25 @@ use crate::scheduler::{Schedule, Scheduler};
 #[derive(Debug)]
 pub struct RoundRobinScheduler {
     iterations: usize,
+    max_iterations: usize,
```

Comment:
@jdeans289 added a new constructor for the `RoundRobinScheduler`, so that it would be possible to use it with the determinism checker (otherwise it would only be possible to test it once). @jamesbornholt then commented that we could probably just combine the two. I dunno, there is no particular reason to use the `RoundRobinScheduler` when checking for uncontrolled randomness, so no big need to change it.

Reply:
Left it in. I can unchange it if you want.